index.html

```diff
@@ -423,18 +423,14 @@ <h3>Firing events using the <code>PointerEvent</code> interface</h3>
                 <p>If the event is {{GlobalEventHandlers/pointerdown}}, the associated device is a direct manipulation device, and the target is an {{Element}},
                     then <a>set pointer capture</a> for this <code>pointerId</code> to the target element as described in <a>implicit pointer capture</a>.</p>
 
-                <p>Fire the event to the determined target.
-                    The user agent SHOULD treat the target as if the pointing device has moved over it from the |previous target|
-                    for the purpose of <a data-cite="uievents/#events-mouseevent-event-order">ensuring event ordering</a> [[UIEVENTS]].
-                    If the |needs over event| flag is set, an over event is needed even if the target element is the same.
-                </p>
-
-                <p>Save the determined target as the |previous target| for the given pointer,
-                    and reset the |needs over event| flag to <code>false</code>.
-                    If the |previous target| at any point will no longer be [=connected=] [[DOM]],
-                    update the |previous target| to the nearest still [=connected=] [[DOM]] parent
-                    following the event path corresponding to dispatching events to the previous target,
-                    and set the |needs over event| flag to <code>true</code>.
+                <p>Fire the event to the determined target.  Before firing this event, the user agent SHOULD treat the target as if the pointing device has moved over it from the |previousTarget| for the purpose of <a data-cite="uievents/#events-mouseevent-event-order">ensuring event ordering</a> [[UIEVENTS]]. If the |needsOverEvent| flag is set, a {{GlobalEventHandlers/pointerover}} event is needed even if the target element is the same.</p>
```

Comment:
Looks good, but maybe we should move the "Fire the event to the determined target" after the stuff to do before firing the event for clarity? I.e.

Before firing this event, the user agent SHOULD treat the target as if the pointing device has moved over it from the |previousTarget| for the purpose of <a data-cite="uievents/#events-mouseevent-event-order">ensuring event ordering</a> [[UIEVENTS]]. If the |needsOverEvent| flag is set, a {{GlobalEventHandlers/pointerover}} event is needed even if the target element is the same.  Fire the event to the determined target.

Or even split them up into separate paragraphs:

<p>Before firing this event, the user agent SHOULD treat the target as if the pointing device has moved over it from the |previousTarget| for the purpose of <a data-cite="uievents/#events-mouseevent-event-order">ensuring event ordering</a> [[UIEVENTS]]. If the |needsOverEvent| flag is set, a {{GlobalEventHandlers/pointerover}} event is needed even if the target element is the same.</p>

<p>Fire the event to the determined target.</p>

Reply:
Done.